tools/nodejs/src/duckdb_node.hpp

```diff
@@ -127,8 +145,8 @@ class Statement : public Napi::ObjectWrap<Statement> {
 	Napi::Value All(const Napi::CallbackInfo &info);
 	Napi::Value Each(const Napi::CallbackInfo &info);
 	Napi::Value Run(const Napi::CallbackInfo &info);
-	Napi::Value Bind(const Napi::CallbackInfo &info);
```

Comment:
I think this is just dead code—there isn't an implementation, there are no callers, and it's not exposed to javascript